### PR TITLE
Initial version of metricbeat

### PR DESCRIPTION
**Pull request by Scriptkiddi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+| `metricbeat_config`  | `{ output.elasticsearch: { hosts: ["localhost:9200"] }, metricbeat.config.modules: { reload.enabled: false } }` | Since the metricbeat settings file is a yaml file we write the whole object to the settings file. For more Information please see the [metricbeat settings file documentation](https://www.elastic.co/guide/en/metricbeat/current/metricbeat-settings-file.html) |
+| `metricbeat_modules` | `[]`                                                                                                                                                 | List of config files to write to modules.d/. More information below.                                                                                                                                                                                     |
+
+### `metricbeat_modules`
+Each entry in the `metricbeat_modules` consists out of the following entries.
```
Comment (SlothOfAnarchy) [inline README.md]: why not use a dict?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+```yml
+- hosts: all
+  become: true
+  vars:
+    metricbeat_config:
+      metricbeat.config.modules:
+        reload.enabled: false
+      output.logstash:
+        hosts: ["localhost:9200"]
+	metricbeat_modules:
```
Comment (SlothOfAnarchy) [inline README.md]: What's with the whitespace? See github web view.

File: tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]: What's with alle the extra lines (in all files)?

File: tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- name: Install apt-transport-https
+  apt: 
+    name: "{{ item }}"
+  with_items:
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]: Why are you using `with_items` here?

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 3 above)
Comment (Scriptkiddi) [reply tasks/main.yml]: fixed

File: tasks/main.yml (hunk 4 above)
Comment (Scriptkiddi) [reply tasks/main.yml]: fixed

File: README.md (hunk 1 above)
Comment (Scriptkiddi) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (Scriptkiddi) [reply README.md]: fixed

**Comment by Scriptkiddi** [comment]
@SlothOfAnarchy pls rereview

File: README.md (hunk 2 above)
Comment (SlothOfAnarchy) [reply README.md]: unfortunately not

File: README.md [hunk 5]
```diff
@@ -2,25 +2,20 @@
...
+| `metricbeat_modules` | `{}`                                                                                                                                                 | Dict of config files to write to modules.d/. More information below.                                                                                                                                                                                     |
 
 ### `metricbeat_modules`
 Each entry in the `metricbeat_modules` consists out of the following entries.
-| Variable | Default / Mandatory | Description                                                                              |
-|----------|---------------------|------------------------------------------------------------------------------------------|
-| `name`   | :heavy_check_mark:  | Name of the module config file. This value has to be the same as the name of the module. |
-| `config` | :heavy_check_mark:  | Yaml config for the module name .                                                        |
+The key of each entry is used to determine the name of the config file and the content of the entry is writen to the config file
```
Comment (SlothOfAnarchy) [inline README.md]: *written
`.` to conclude the sentence

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

**Comment by Scriptkiddi** [comment]
@SlothOfAnarchy pls rereview

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**